packages/integration-karma/test/component/LightningElement.toString/index.spec.js

```diff
@@ -3,14 +3,14 @@ import { createElement } from 'lwc';
 import Anonymous from 'x/anonymous';
 import Named from 'x/named';
 
-// TODO [#1033]: with the transformation done by the compiler, the class name is removed, producing "[object undefined]"
-xit('should rely on the constructor name', () => {
+it('should rely on the constructor name', () => {
     const elm = createElement('x-named', { is: Named });
     expect(elm.getToString()).toBe('[object MyFancyComponent]');
 });
 
-// TODO [#1033]: open issue return "[object ]" on Safari
-xit('should fallback to BaseLightningElement if constructor has no name', () => {
-    const elm = createElement('x-anonymous', { is: Anonymous });
-    expect(elm.getToString()).toBe('[object BaseLightningElement]');
-});
+if (process.env.COMPAT !== true) {
+    it('should fallback to BaseLightningElementConstructor if constructor has no name', () => {
```

Comment:
The constructor name ends up as `_class` in compat mode. This seems fine, given that it's no worse than the `undefined` that we render today.

Reply:
I'm ok with that. Who cares about compat :)?